app/graphql/types/query_type.rb

```diff
@@ -11,7 +11,13 @@ class QueryType < Types::BaseObject
       argument :order_by, Types::ProviderOrder, required: false
     end
     def providers(filters: nil, order_by: nil)
-      ::Provider.where(where(filters)).order(order(order_by))
+      filters = filters.to_h
```

Comment:
Believe I have to transform into hash here in order to use `delete` method?

Reply:
That may be true, yes.